pkgs/development/python-modules/backports-asyncio-runner/default.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  hatchling,
+  hatch-fancy-pypi-readme,
+}:
+
+let
+  pname = "backports-asyncio-runner";
+  version = "1.2.0";
+in
+buildPythonPackage {
+  inherit pname version;
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "samypr100";
+    repo = "backports.asyncio.runner";
+    tag = "v${version}";
+    hash = "sha256-F8x7MZgu0VItH7kBke7C7+ZBoM6Iyj8xOeQ2t56ff3k=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+    hatch-fancy-pypi-readme
+    hatchling
+  ];
+
+  # Disabling tests due to "ModuleNotFoundError: No module named test.test_asyncio"
+  doCheck = false;
```

Comment:
Did you use `pytestCheckHook` or `unittestCheckHook`?
Can you post the entire error log?

Reply:
You add use `disabledTestPaths` instead of `doCheck = false`. Please add the explanation of where test.test_asyncio would come from to your inline comment.